app/assets/stylesheets/application.scss

```diff
@@ -44,3 +44,23 @@ $govuk-typography-use-rem: false;
 .release__view-diff {
   float: right;
 }
+
+.non-production-version {
+  @include govuk-font(16);
+  margin: 0 15px 15px 15px;
+  border: 1px solid #B01117;
+  background-color: govuk-colour('red');
+  color: govuk-colour('white');
+  padding: 1em;
+  font-weight: bold;
+
+  a {
```

Comment:
instead of this, please add the class .govuk-link to the a tag, then have a selector of .release__non-production-banner .govuk-link to override the colour and make it white. You should get the rest for free.